### PR TITLE
Fix #2679 - Cesium Inspector click events for IE

### LINKED ISSUE #2679: Cesium Inspector does not work in IE
Neither the "Pick a Primitive" or "Pick a Tile" functions work in the Cesium Inspector in IE. Apparently this has been broken for a while (assuming it ever worked at all).

### PR DESCRIPTION
**Pull request by hpinkos**
#2679

### CODE REVIEW

File: Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js [hunk 1]
```diff
@@ -405,9 +407,19 @@ define([
         this._pickPrimitive = createCommand(function() {
             that.pickPrimitiveActive = !that.pickPrimitiveActive;
             if (that.pickPrimitiveActive) {
-                that._canvas.addEventListener('mousedown', pickPrimitive, false);
+                if (FeatureDetection.supportsPointerEvents()) {
+                    that._canvas.addEventListener('pointerdown', pickPrimitive, true);
+                } else {
+                    that._canvas.addEventListener('mousedown', pickPrimitive, true);
+                    that._canvas.addEventListener('touchstart', pickPrimitive, true);
```
Comment (emackey) [inline Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js]: `touchstart` doesn't return `e.clientX`, it has a list of touches with a clientX per touch.  I think each API needs to be handled separately here, unless you switch to Cesium's built-in event handling.

**Comment by hpinkos** [comment]
Thanks @emackey! I forgot about touch events.
I switched it to use the `ScreenSpaceEventHandler` instead.

**Comment by emackey** [comment]
This looks good.  Question, the new `eventHandler` is never destroyed, should it be?  Is it even possible to destroy the Cesium Inspector itself?

**Comment by emackey** [comment]
Cesium Inspector is such a niche debugging tool, maybe we don't care about destroying it properly.  Still, the old `viewerEntityMixin` used to [wrap the Viewer destructor](https://github.com/AnalyticalGraphicsInc/cesium/blob/1.1/Source/Widgets/Viewer/viewerEntityMixin.js#L301-L319) for proper cleanup, maybe it's worth adding such a wrapper to Inspector?  You wouldn't need to "removeInputAction" like this old code, just destroy the new event handler.

**Comment by hpinkos** [comment]
@emackey there's a [destroy on the CesiumInspector](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Widgets/CesiumInspector/CesiumInspector.js#L333-L338) class, just not on the view model.  Should I pipe it through to destroy the event handler?

**Comment by emackey** [comment]
If it's not a big deal, yes.  If the rest of the team thinks there are bigger fish to fry, I'll defer to them.

**Comment by hpinkos** [comment]
@emackey no big deal, took like two seconds.  is this better?

**Comment by emackey** [comment]
Great!  Tests pass, original issue appears fixed.  Merging.

**Pull request merged**